.travis.yml

```diff
@@ -10,6 +10,7 @@ python:
 cache: pip
 
 before_install:
+  - sudo add-apt-repository -y ppa:jonathonf/texlive-2016
```

Comment:
Could you clarify why this is needed / what error is fixed by this? Thanks!

Reply:
These are Sphinx errors, so they're not going to be affected by updating texlive.

Reply:
OK!